### ISSUE TITLE
Handle channels and webhooks set by users as strings instead of string lists and the null default value

### ISSUE THREAD

**Issue by dominik-korsa** [dominik-korsa/discord-integration#42]
While the `config.yml` in resources.yml looks like this:
```yml
chat:
  channels:
  # - 0123456789
  # - 01248163264128256
  webhooks:
  # - https://discord.com/api/webhooks/...
  ```
The one generated by Boosted YAML is actually (notice the `null`):
 ```yml
chat:
  channels: null
  # - 0123456789
  # - 01248163264128256
  webhooks: null
  # - https://discord.com/api/webhooks/...
  ```
  
Users often incorrectly place the string values in place of `null`, like this:
```yml
chat:
  channels: 43219876

  webhooks: discord.com/api/webhooks/...

  ```

**Comment by RzMagix** [comment]
What Would Be The Correct Way To Format It

**Comment by dominik-korsa** [comment]
It should instead look like this:
```yml
chat:
  channels: 
  - 43219876

  webhooks:
  - discord.com/api/webhooks/...

  ```

**Comment by dominik-korsa** [comment]
Converting strings to string lists has been fixed in 1e8179b67beca376a816e9245549d2d2c3d43b75, but the default values still appear as null and also the comments are stripped
